calico-enterprise_versioned_docs/version-3.16/threat/honeypod/honeypods.mdx

```diff
@@ -40,10 +40,16 @@ Honeypods can be configured on a per-cluster basis using "template" honeypod man
 
 ### Configure namespace and RBAC for honeypods
 
+In a v1.24 or earlier cluster, if the [PodSecurityPolicy admission controller](https://v1-24.docs.kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy) is enabled, install the required Honeypod Pod Security Policy:
+
+```bash
+kubectl create -f {{filesUrl}}/manifests/threatdef/honeypod/psp-honeypod.yaml
```

Comment:
@hjiawei The build is failing because this file doesn't exist. Do you know what's going on with this psp-honeypod.yaml file in 3.16?

Reply:
I see from comments elsewhere that this file should be added for 3.16.2. Assuming that still holds true, we'll need to remove the CC changes from this PR. I'll track the changes for CC and add them when the core CC version is updated, which will be after CE 3.16.2 is released.

FYI @bmckercher123

Reply:
CC changes are reverted.

Reply:
@hjiawei Thanks. We'll wait for the next patch to go out, then get this merged.